src/validator/test/QueryValidatorTest.cpp

```diff
@@ -1089,6 +1089,18 @@ TEST_F(QueryValidatorTest, GoInvalid) {
         std::string query = "GO FROM \"2\" OVER like YIELD COUNT(123);";
         EXPECT_FALSE(checkResult(query));
     }
+    {
+        std::string query = "GO FROM \"1\" OVER like YIELD like._dst AS id, like._src AS id | GO "
+                            "FROM $-.id OVER like";
+        auto result = checkResult(query);
+        EXPECT_EQ(std::string(result.message()), "SemanticError: Duplicate Column Name : `id'");
+    }
+    {
+        std::string query = "$a = GO FROM \"1\" OVER like YIELD like._dst AS id, like._src AS id; "
+                            "GO FROM $a.id OVER like";
+        auto result = checkResult(query);
+        EXPECT_EQ(std::string(result.message()), "SemanticError: Duplicate Column Name : `id'");
+    }
```

Comment:
Add a case about define duplicate alias but not used in latter sentence.

Reply:
ok, will add case in next pr

Reply:
Ok.